src/Type/Comment/Connection/CommentConnectionDefinition.php

```diff
@@ -35,7 +35,7 @@ class CommentConnectionDefinition {
 	 */
 	public static function connection() {
 
-		if ( null === self::$connection ) {
+		if ( null === self::$connection ) :
```

Comment:
Would be good to get a developers handbook going for rules on stuff like this. When to use `:` vs `{`. Or at least an issue so we can start documenting these things.